templates/cluster/aws-hosted-cp/templates/k0smotroncontrolplane.yaml

```diff
@@ -53,6 +62,31 @@ spec:
             chartname: aws-ebs-csi-driver/aws-ebs-csi-driver
             version: 2.33.0
             values: |
+              image:
+                repository: {{ .Values.extensions.imageRepository }}
+              sidecars:
+                provisioner:
+                  image:
+                    repository: {{ .Values.extensions.imageRepository }}
+                attacher:
+                  image:
+                    repository: {{ .Values.extensions.imageRepository }}
+                snapshotter:
+                  image:
+                    repository: {{ .Values.extensions.imageRepository }}
+                livenessProbe:
+                  image:
+                    repository: {{ .Values.extensions.imageRepository }}
+                resizer:
+                  image:
+                    repository: {{ .Values.extensions.imageRepository }}
+                nodeDriverRegistrar:
+                  image:
+                    repository: {{ .Values.extensions.imageRepository }}
+                volumemodifier:
+                  image:
+                    repository: {{ .Values.extensions.imageRepository }}
```

Comment:
We should use conditionals when using `.Values.extensions.imageRepository`, since by default it's an empty value. We don't want any empty values rendered.

Applicable to all uses of `.Values.extensions.imageRepository` and `.Values.extensions.chartRepository`

Reply:
`chartRepository` already is wrapped with conditionals, I've updated to wrap all instances of `imageRepository as well.